apis/Google.Cloud.Logging.Log4Net/docs/configuration.md

```diff
@@ -123,7 +127,24 @@ If `resourceType` has been manually set, then `resourceLabel` is used to specify
 
 Defaults to `false`. Setting this to `true` disables automatic `resourceType` setting based on platform detection.
 
-### Complete configuration example
+### CredentialFile
+
+```xml
+<credentialFile value="path/to/credential.json" />
+
+Specifies the path to a JSON file containing credentials.
```

Comment:
Do we need to close the xml code snippet? ```

Reply:
Doh, yes thanks!